src/main/java/domain/ladder/Ladder.java

```diff
@@ -0,0 +1,44 @@
+package domain.ladder;
+
+import domain.ladder.exception.InvalidLadderHeightException;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Ladder {
+    private static final int MIN_HEIGHT = 1;
+
+    private final List<LadderLine> ladderLines;
+
+    public Ladder(final List<LadderLine> ladderLines) {
+        validHeight(ladderLines.size());
```

Comment:
```suggestion
        validHeight(size());
```
아래 ladderLines.size()를 호출하고 있는 size() 메서드를 호출하는건 어떤가요? 추가로 size라는 메서드 이름보다 height라는 메서드명은 어떤가요?

Reply:
짜잔~ ladderLine.size()는 매개변수였습니다.

Reply:
height라는 메서드명이 더 보기 좋겠네요 ^^